build.sh

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env bash
+declare -A cht_versions=(
+  ["cht-core-4-5"]="15bd199eff7468b23acae5e39b2245ce4b4a5220" # Commit that added cht-form (technically after of 4.5, but before the Enekto uplift in 4.6)
```

Comment:
Do we want to include this?  It contains some code that was added after `4.5` (since the cht-form functionality was added after `4.5`, but it does not include the Enketo uplift coming in `4.6` so it might be useful in determining if a form problem is a result of the new Enekto version...

Reply:
> Sounds like it'd be mostly useful for core devs?

Yeah, this is a good point and probably the general reason I was thinking this might be useful.  (Though one of my other main goals here was just to demonstrate multiple CHT versions in the harness.)  With all this in mind, I will go ahead and remove support for `cht-core-4-5` as it is probably not useful and could add unnecessary confusion!